### PR TITLE
feat: run tests on circleci

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

**Comment by moliva** [comment]
I see a `Too long with no output (exceeded 10m0s): context deadline exceeded` possibly from CircleCI. We could change the value to something bigger as we know they take time, as in:
```
- run:  
    command: ...
    no_output_timeout: 30m
```
See: https://support.circleci.com/hc/en-us/articles/360007188574-Build-has-hit-timeout-limit

Reference: https://circleci.com/gh/decentraland/explorer/3223

File: .circleci/config.yml [hunk 1]
```diff
@@ -366,6 +419,13 @@ workflows:
               only: /(.*)/
             tags:
               only: /^\d+\.\d+\.\d(.*)/ # npm tag + latest
+      - run-unity-tests:
+          <<: *all_branches_and_tags
+          requires:
+            - build-unity
+          filters:
+            branches:
+              only: /(master)|(release)|(staging)|(feat\/runtest)/
```
Comment (moliva) [inline .circleci/config.yml]: IMO, we should change this to run in every branch if possible.
```suggestion
              only: /(.*)/
```

**Comment by eordano** [comment]
Running on all branches is not feasible economically (40 runs are 150 usd).

**Comment by eordano** [comment]
image
Ready to merge :D

File: unity-client/Assets/Scripts/Tests/CharacterControllerTests/CharacterControllerTests.cs [hunk 2]
```diff
@@ -133,6 +133,7 @@ public IEnumerator Character_UpdateSOPosition()
         }
 
         [UnityTest]
+        [NUnit.Framework.Explicit("fixme (@pravus)")]
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/Tests/CharacterControllerTests/CharacterControllerTests.cs]: Please add `[Category("Explicit")]` so we can check the explicit tests easily via the Test Runner editor window.

**Review by BrianAmadori — APPROVED** [review]

File: unity-client/Assets/Scripts/Tests/CharacterControllerTests/CharacterControllerTests.cs (hunk 2 above)
Comment (eordano) [reply unity-client/Assets/Scripts/Tests/CharacterControllerTests/CharacterControllerTests.cs]: ouch, this slipped me, thanks

File: .circleci/config.yml [hunk 3]
```diff
@@ -366,6 +413,13 @@ workflows:
               only: /(.*)/
             tags:
               only: /^\d+\.\d+\.\d(.*)/ # npm tag + latest
+      - run-unity-tests:
```
Comment (moliva) [inline .circleci/config.yml:416]: Normally I would ask the publish renderer step to require this to pass, maybe we can add this in the short future after checking that tests are running smoothly.

**Review by moliva — APPROVED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (eordano) [reply .circleci/config.yml:416]: adding it!

**Pull request merged**